GetIntoTeachingApi/Models/Crm/CandidateQualification.cs

```diff
@@ -14,6 +14,16 @@ public enum DegreeType
             Degree = 222750000,
             DegreeEquivalent = 222750005,
         }
+        
+         public enum UkDegreeGrade
+        {
+            LowerSecond = 222750003,
```

Comment:
could you move this to line 24 so they are ordered by ID - it will help in future if we need to add new items